dists/emscripten/emscripten-15893-fix.patch

```diff
@@ -0,0 +1,44 @@
+From 2ea11b4734de5656aaf3285145831e2bb9ad53ae Mon Sep 17 00:00:00 2001
```

Comment:
scummvm repo is not good place to keep distro patches

Reply:
I agree this is a hack. Is it general not a good idea to include patches (and instead rely on only stable versions), or is this particular one a blocker since it isn't even in the PR of the upstream project? 

If it's the second, I would try to get this added to PR emscripten-core/emscripten#15893 (or start a second PR), so all patches have a corresponding PR. If we in general don't want to rely on unreleased code, this will mean:

- Either keep this PR open until all changes are merged and released in upstream
- Accept that certain things wont work with the merged code without manually applying these patches (this can be documented and a second PR could be put in draft until the changes all have landed)

The reason that these patches are checked into this repo, is because PRs aren't 100% stable (they are often rebased against master, which means they might not work against the stable release this script is depending on)

Reply:
you can create upstream forks with patches applied and point to them. or put patches somewhere on GitHub and point to them

Reply:
I cleaned up the patches in 037536e8440e7931cdf62d1a0bf265e73ea644c1. All patches are now downloaded from Github (I still checked them in as well as PRs aren't 100% stable which could break the build).

Reply:
I still see patches in PR

Reply:
I wasn't aware that patches in general aren't allowed. I removed them and uploaded them to another repo (to ensure they stay stable and don't change should the PRs change)